.github/workflows/python-app.yml

```diff
@@ -15,7 +15,14 @@ permissions:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # MacOS of interest, ver. 11 or Big Sur, seems not available on GHA
+	# so using the one close as possible.
+        os: [debian-12, ubuntu-24.04, macos-13-large]
+    continue-on-error: ${{ matrix.os == 'macos-13-large' }}
```

Comment:
As reported in https://github.com/kinu-garage/essay_in_idleness/issues/380, the entire pipeline is marked as failure due to the MacOS' job failure, so this configuration may not be working.